addon/services/rollbar.js

```diff
@@ -5,56 +5,63 @@ import deepMerge from 'lodash.merge';
 import Rollbar from 'rollbar';
 
 export default Service.extend({
-  enabled: computed({
+  _rollbar: null,
+
+  enabled: computed('config.enabled', {
```

Comment:
See [require-computed-property-dependencies](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/require-computed-property-dependencies.md)

Reply:
That was intentional :) We should revert.